src/background/utils/notifications.js

```diff
@@ -1,18 +1,20 @@
-import { i18n, defaultImage, sendTabCmd } from '#/common';
+import { defaultImage, sendTabCmd } from '#/common';
 import { commands } from './message';
 
 const openers = {};
 
 Object.assign(commands, {
   /** @return {Promise<string>} */
-  async Notification(data, src, bgExtras) {
+  async Notification(data, src, bgExtras = {}) {
+    const { items, onClick, type } = bgExtras;
     const notificationId = await browser.notifications.create({
-      type: 'basic',
-      title: data.title || i18n('extName'),
+      type: type || 'basic',
+      title: data.title || '',
```

Comment:
I removed the default title because Chrome/FF already displays "Violentmonkey" in the non-customizable header of the notification.